backend/app/services/chat/storage/task_manager.py

```
@@ -636,6 +636,57 @@ async def create_task_and_subtasks(
+            memory_manager.save_user_message_async(
+                user_id=user.id,
+                team_id=team.id,
+                task_id=task.id,
+                subtask_id=user_subtask.id,
+                message=message,
+                workspace_id=workspace_id,
+                group_id=None,  # Future: extract from task spec
+                is_group_chat=is_group_chat,
+                model_config=model_config,  # Pass model config
+            )
+        )
```

Ruff (0.14.11)

Store a reference to the return value of `asyncio.create_task`

(RUF006)